sycl/doc/extensions/proposed/sycl_ext_intel_dataflow_pipes.asciidoc

```diff
@@ -174,19 +188,21 @@ The pipe class exposes static member functions for writing a data word to a pipe
 
 Blocking and non-blocking forms of the read and write members are defined, with the form chosen based on overload resolution.
 
+The `sycl::memory_order` parameter of read/write functions controls how other memory accesses, including regular, non-atomic memory accesses, are to be ordered around the pipe read/write operation.  The default memory order is `sycl::memory_order::seq_cst`.
```

Comment:
I believe we're still discussing the best default.  While seq_cst is more consistent with C++ atomics, it is a change from how pipes work today, which is equivalent to memory_order::relaxed.  And SYCL 2020 atomic_ref uses a third approach where the default must be specified as a template parameter of the class itself.  I think it's fine to leave this sentence in for now, but can you add an "opens" section at the bottom of the extension and mention the choice of default as an open?

Reply:
@pcolberg Added to Issues section, so this can be closed too.